src/_companies/europe/uk/london/gofreerange/index.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Go Free Range
+
+continent: "Europe"
+country: "UK"
+region: "London"
+city: "London"
+
+website: https://gofreerange.com/
+careers_page: https://gofreerange.com/
+
+stack:
+  - "rails"
+
+created_at: 2023-03-20 10:24:17
+---
+
+Founded in 2009, Go Free Range is a worker co-operative with some of the most experienced application developers in the UK. We pair our extensive development experience with the complementary non-development skills necessary to give software projects the best chance of success. As worker-owners we have a vested interest in your project: success for you means success for us.
```

Comment:
Could you change the description to be written in the first person? This way, we can keep all the descriptions consistent.

Reply:
@CalvinWalzel I read the descriptions for the other companies and realised they are all written in the 3rd person, so I've changed ours to use that - I hope that makes sense!

Reply:
That's what I meant, sorry for the confusion 🤦 
Thanks for the adjustment!